test/integration/sagemaker/test_gluonnlp.py

```diff
@@ -0,0 +1,40 @@
+# Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
I don't think this PR build will run this remote integration test -- has this been run on a built container yet?

Reply:
It is run. You can check the log above: https://6gf6v5l0wc.execute-api.us-west-2.amazonaws.com/Prod/buildlogs?key=90024de7-dbb3-4add-8ad9-8394ea41e40c%2Fbuild.log